sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/AggregateDocumentQueryExecutionContext.java

```diff
@@ -20,7 +20,7 @@
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
-import java.util.function.Function;
+import java.util.function.BiFunction;
```

Comment:
Please update title with more user context: Like what scenario/state user sees.

Reply:
updated to ClassCastExceptionForTopQueryWithAggregate~